woocommerce/woocommerce-ios
# [Analytics Hub] Add loading state

Part of https://github.com/woocommerce/woocommerce-ios/issues/8189.

## Description

This PR adds loading state to the Analytics Hub.

## Testing

1. Build and run the app in debug/alpha mode.
2. On the dashboard screen tap the "See more" button.
3. On analytics hub screen observe data loading animation.
4. After loading finished confirm that UI updates with correct data.
5. Go back to dashboard and turn networking off.
6. Tap the "See more" button.
7. Wait for network timeout (1 min).
8. Observe that screen switched to empty state.

## Video

https://user-images.githubusercontent.com/3132438/204596006-d79d2d48-6256-47d2-b850-2ae834f65e93.mp4

---
